npm/metrics/ai-utils.go

```diff
@@ -83,7 +83,11 @@ func SendMetric(metric aitelemetry.Metric) {
 
 // SendLog sends log
 func SendLog(operationID int, msg string, printLog bool) {
-	msg = fmt.Sprintf("%s - (NPM v%d)", msg, npmVersion)
+	osName := "linux"
+	if util.IsWindowsDP() {
+		osName = "windows"
+	}
+	msg = fmt.Sprintf("%s - (NPM v%d) - (%s)", msg, npmVersion, osName)
```

Comment:
have a different identifier, so that it will be easy to add a regex to query this out in kusto

Reply:
may be create a json and log it, so it will be easier to use parse_json in kusto

Reply:
We have multiple values in messages db which point to windows. Like Auth id column gives the OS name, custom dimensions have the os version etc. So adding this might be redundant